Source/WebKit/Platform/cocoa/WKMaterialHostingSupport.swift

```diff
@@ -112,12 +120,32 @@ private struct MaterialHostingView<P: MaterialHostingProvider>: View {
         self.cornerRadius = cornerRadius
     }
 
+    #if os(macOS)
+    func updateAccessibilityState() {
+        shouldIncreaseContrast =
+            NSWorkspace.shared
+            .accessibilityDisplayShouldIncreaseContrast
+        shouldReduceTransparency = NSWorkspace.shared.accessibilityDisplayShouldReduceTransparency
+    }
+    #endif
+
     var body: some View {
         let view = P.view(for: content)
 
         if let effect = MaterialHostingView<P>.resolvedMaterialEffect(for: materialEffectType) {
             AnyView(view.materialEffect(effect, in: .rect(cornerRadius: cornerRadius)))
                 .environment(\.colorScheme, colorScheme == .light ? .light : .dark)
+                #if os(macOS)
+            .environment(\._accessibilityReduceTransparency, shouldReduceTransparency)
```

Comment:
I noted this in the commit message too, this indentation is applied by `swift-format`.